run-tests.sh

```diff
@@ -7,7 +7,7 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 pydocstyle reana_client && \
-isort -rc -c -df **/*.py && \
+#isort -rc -c -df **/*.py && \
```

Comment:
We are seeing this problem everywhere, we should fix it and propagate the change to the rest of problematic repos. @okraskaj did you fix it already? If so please share with @dinosk.